src/content/basics/secured-access-to-clusters/index.md

```diff
@@ -0,0 +1,81 @@
++++
+title = "Secured access to clusters for users and Giant Swarm support"
+description = "This documentation explains security measures for users and Giant Swarm support accessing offered services"
+date = "2019-05-09"
+weight = 45
+type = "page"
+categories = ["basics"]
++++
+
+# Secured access to clusters for users and Giant Swarm support
+
+In order to provide the best service possible besides you having access to your clusters, also Giant Swarm staff will be able to connect.
+
+Thus it should be explained how this access differs and what security measures are brought into place so that your data is saved with us. 
+
+## Intro
+
+Access to Giant Swarm clusters can be split into two parts. 
+
+1. Admin Access - designated for Giant Swarm staff for managing/development/support purposes.
+
+2. User Access - designated for Giant Swarm customers to interact with the offered services.
+
+If you would like to know more on the different parts of the Giant Swarm infrastructure, please see our [operational layers article](/basics/giant-swarm-operational-layers/)
+
+## Admin access
+
+Admin access is guarded by Virtual Private Network (VPN) that is managed via certificates (public/private keys) and secured connection between both VPNs.
+
+Certificates management is handled by one of Giant Swarm's private components. The general principle is that a certificate is configured for each individual Giant Swarm staff member.
+
+VPN secured access points:
+
+* **SSH** - SSH Admin access is based on the Github SSO. Only users residing in the Github Giant Swarm Organization are allowed to authenticate. Following process describes more in detail the procedure of SSH authentication:
+
+![](./ssh_access_process.png)  
+
+  SSH access to Control Plane allows Giant Swarm also to manage and connect to underlying Data Platforms of the Customer.
```

Comment:
What are 'Data Platforms'?

Reply:
in fact that is Tenant Cluster, i found Data Platform phrase in one of the architecture diagrams in giantswarm repo